code/online_challenges/src/codechef/BFRIEND/Bfriend.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <set>
+typedef long long int ll;
+
+void solve() {
+    ll n, i, a, b, c, ans1, ans2;
+    std::set<ll> ss;
+    ll x;
+    std::cin >> n >> a >> b >> c;
+    for (i = 0; i < n; ++i) {
+        std::cin >> x;
+        ss.insert(x);
+    }
+    if (n == 1) {
+        auto it = ss.begin();
+        ans1 = abs((*it) - a) + c + abs((*it) - b);
+        std::cout << ans1 << "\n";
```

Comment:
Instead of just printing out the value in this function, return it and print it out in `main`.